griptape/drivers/vector/pinecone_vector_store_driver.py

```diff
@@ -34,7 +34,7 @@ def upsert_vector(
 
         params = {"namespace": namespace} | kwargs
 
-        self.index.upsert([(vector_id, vector, meta)], **params)
+        self.index.upsert([(vector_id, vector, meta)], str(params))
```

Comment:
Is this right? ** generates a dictionary which we are stringifying?

Reply:
Yeah this change should be reverted. Curious what the pyright error was?

Reply:
This was 2 pyright errors:
`Argument of type "str | None" cannot be assigned to parameter "batch_size" of type "int | None" in function "upsert"` and `Argument of type "str | None" cannot be assigned to parameter "show_progress" of type "bool" in function "upsert"`